1config-cli/src/com/brunobonacci/oneconfig/main.clj

```diff
@@ -194,6 +194,7 @@ NOTE: set AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY or AWS_PROFILE to
        (#{:get :set :init :list :list-keys :create-key} op)) (help! ["INVALID operation: must be either GET, SET, LIST, INIT, LIST-KEYS or CREATE-KEY"])
       (and (= op :set) (not (or value content-file))) (help! ["MISSING: required argument: value, provide a value in-line or via the '-f filename' option"])
       (and (= op :create-key) (not master-key)) (help! ["MISSING: required argument: master-key"])
+      (and (or (= op :get) (= op :set)) (some nil? [env key version])) (help! [str "MISSING: required arguments: " (util/required-names {:env env, :key key, :version version})])
```

Comment:
not too sure here about `(util/required-names {:env env, :key key, :version version})`
if you call the function with the following parameters:
```
(util/required-names {:env "dev", :key nil, :version nil"})
;;=> "env"
```
while you probably want the opposite, a message which says:

```
MISSING: required arguments: key, version
```